db/institutions.yml

```diff
@@ -103,6 +103,14 @@ bnl:
   name: Brookhaven National Laboratory
   state: NY
   zip: 11973
+esrf:
+  aka: [European Synchrotron Radiation Facility, ESRF]
+  city: Grenoble
+  country: France
+  departments: {dubble: Dutch-Belgian Beamline}
```

Comment:
I am not sure that this key is intuitive.  Maybe `db-bl`?
This has the has the advantage that we can have more beamlines that are `<sthg>-bl`

Reply:
Agreed. I will also add ``aka`` as ``sas-bl``.